Готова 7 проектная работа

SmithDaria opened the pull request:
Отправляю на ручное ревью

SmithDaria committed:
Готова 7 проектная работа

yana-yanchenko commented on index.html:
```diff
@@ -1,18 +1,24 @@
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="./styles/animations.css">
+    <link rel="stylesheet" href="./styles/themes.css">
   </head>
```
head оформлен верно

yana-yanchenko commented on index.html:
```diff
@@ -1,18 +1,24 @@
-    <header class="header">
+    <header class="header section-center">
+      <a class="header__logo">&lt;/HTML&gt;</a>
+      <h1 class="header__title">закрывающий тег</h1>
```
верный уровень заголовок

yana-yanchenko commented on index.html:
```diff
@@ -39,8 +45,80 @@
+        </div>
+      </article>
+    </main>
+    <footer class="footer section-center">
```
надо исправить

Вложенное содержимое не представляет подвал страницы, тег `<footer>` следует удалить
Содержимое необходимо перенести в` <main>`

yana-yanchenko commented on index.html:
```diff
@@ -39,8 +45,80 @@
+        <svg width="21" height="21" viewBox="0 0 21 21" class="save-icon">
+          <use href="/svg/floppy.svg#floppy"></use>
+        </svg>
+        <span class="button__text">Сохранить на память</span>
```
можно лучше 

Хорошо бы добавить между предлогом и словом добавить неразрывные пробелы, чтобы не было повисших предлогов. https://www.artlebedev.ru/typograf/

yana-yanchenko commented on styles/variables.css:
```diff
@@ -1 +1,32 @@
+  --normal-font-weight: 400;
+  --bold-font-weight: 700;
+  --background-image: url(../svg/pattern-dark.svg);
+}
```
надо исправить

в файле variadles следует хранить только общий css переменные

yana-yanchenko commented on styles/animations.css:
```diff
@@ -12,3 +12,89 @@
+    scale: 1.5;
+  }
+}
+.like-icon.is-liked .heart {
```
Значение для `animation-direction` необходимо указать как `alternate`(проигрывание анимации в обратную сторону после завершения каждого цикла.), подробнее => https://doka.guide/css/animation-direction/

yana-yanchenko commented on styles/animations.css:
```diff
@@ -12,3 +12,89 @@
+  }
+}
+.like-icon.is-liked .sparks {
+  animation: sparks 0.3s 0.3s ease-in;
```
Значение для `animation-direction` необходимо указать как `alternate`(проигрывание анимации в обратную сторону после завершения каждого цикла.), подробнее => https://doka.guide/css/animation-direction/

yana-yanchenko commented on styles/style.css:
```diff
@@ -13,8 +13,220 @@
+  font-family: var(--main-font-family), sans-serif;
+  font-variation-settings: "wght" var(--normal-font-weight);
+  background-image: var(--background-image);
+  background-attachment: fixed;
```
верное решение фиксировать фон

yana-yanchenko commented on styles/style.css:
```diff
@@ -13,8 +13,220 @@
+}
+.theme-menu__button:hover {
+  opacity: 60%;
+  transition: opacity 0.3s ease;
```
Надо исправить

Правила группы `transition` будет лучше один раз указать для основного селектора, тогда переход будет работать на элементе всегда

yana-yanchenko commented on styles/style.css:
```diff
@@ -13,8 +13,220 @@
+.theme-menu__button:focus {
+  outline: none;
+  opacity: 0.6;
+  transition: 0.3s ease;
```
Правила группы `transition` будет лучше один раз указать для основного селектора, тогда переход будет работать на элементе всегда

yana-yanchenko commented on styles/style.css:
```diff
@@ -13,8 +13,220 @@
+}
+.button:hover {
+  color: var(--background-color);
+  transition: color 1s ease;
```
надо исправить

Правила группы `transition` будет лучше один раз указать для основного селектора, тогда переход будет работать на элементе всегда

yana-yanchenko commented on styles/style.css:
```diff
@@ -13,8 +13,220 @@
+.button:focus-visible {
+  box-shadow: 2px 2px 0px 0px var(--accent-color);
+  border: 2px solid var(--accent-color);
+  transition: box-shadow 0.3s ease;
```
надо исправить

Правила группы `transition` будет лучше один раз указать для основного селектора, тогда переход будет работать на элементе всегда

yana-yanchenko reviewed (commented):
Необходимо развернуть общий комментарий ⬇️

Добрый день. Работа выполнена на хорошем уровне! 

Отлично:

* Размечена вся структура
* Используете семантичные теги для разметки.
* аккуратно оформленный код
* все комментарии "Отлично"

Надо исправить:

* Все комментарии с пометкой "надо исправить"
* Проект необходимо выложить на `gh pages`

Можно лучше:

* Все комментарии "Можно лучше" в коде.

Необходимо исправить все "надо исправить". Исправление "можно лучше" остаются на ваше усмотрение.

SmithDaria committed:
Отправляю 7 проект на повторное ревью

Комментарий на счет публикации ввел в смущение, так как в рекомендациях к первой части проекта этого не сказано. Насколько поняла, что именно финальный вариант (вторую часть) необходимо будет опубликовать.

SmithDaria commented:
Комментарий на счет публикации ввел в смущение, так как в рекомендациях к первой части проекта этого не сказано. Насколько поняла, что именно финальный вариант (вторую часть) необходимо будет опубликовать.

yana-yanchenko commented on styles/variables.css:
```diff
@@ -1 +1,31 @@
+  --heart-sparks: #ff0000;
+  --normal-font-weight: 400;
+  --bold-font-weight: 700;
+}
```
надо исправить

в файле variadles следует хранить только общий css переменные

yana-yanchenko reviewed (commented):
Отлично! Работа принята! Успехов в дальнейшем обучении!
